mubench.reviewsite/templates/result_stats.phtml

```diff
@@ -26,7 +26,7 @@ use MuBench\ReviewSite\Model\RunsResult;
 <body>
 <?php include "includes/menu.phtml" ?>
 
-<h1>Statistics</h1>
+<h1>Dataset - Statistics</h1>
```

Comment:
This is the template for the results statistics, right? If so, please name it accordingly, e.g., "Statistics: Results". Be consistent across the different statistics pages.